systemtest/src/main/java/io/strimzi/systemtest/logs/LogCollector.java

```diff
@@ -212,13 +213,13 @@ private void collectReplicaSets(String namespace) {
 
     private void collectStrimzi(String namespace) {
         LOGGER.info("Collecting Strimzi in Namespace {}", namespace);
-        String crData = cmdKubeClient(namespace).exec(false, false, "get", "strimzi", "-o", "yaml", "-n", namespaceFile.getName()).out();
+        String crData = cmdKubeClient(namespace).exec(false, StandardLevel.DEBUG, "get", "strimzi", "-o", "yaml", "-n", namespaceFile.getName()).out();
```

Comment:
Are you sure StandardLevel.DEBUG is the right thing to use? Javadoc for this package suggests it is _Internal interfaces and classes to be used by authors of logging implementations or for internal use by API classes._ I would expect this https://logging.apache.org/log4j/2.x/log4j-api/apidocs/index.html should be used instead?